src/main/java/com/jcabi/log/MulticolorLayout.java

```diff
@@ -183,61 +151,46 @@ public void setColors(final String cols) {
      * @param lev JavaScript like map of levels
      * @since 0.9
      */
+    @SuppressWarnings("PMD.UseConcurrentHashMap")
```

Comment:
@mkordas PMD complains about I'm not using ConcurrentHashMap. The solution I found was that.